Add community translation (#25)
gatsbyjs/gatsby-pt-BR: docs/contributing/community.md

---
title: Comunidade
---

> Desde Setembro de 2019, a comunidade Gatsby é composta por mais de 2.400 colaboradores e dezenas de milhares de desenvolvedores construindo websites pessoais e comerciais usando Gatsby.

Reviewer: Acho que poderia traduzir o termo "websites" para "sites".

_Open source_ nem sempre tem a melhor reputação de ser amigável e acolhedor, e isso nos deixa tristes. **Todos que pertencem ao _open source_, e todos do Gatsby estão dedicados a fazer você se sentir acolhido.**

Nós nunca julgaremos, seremos condescendentes ou excluiremos alguém. Ao invés disso, nós iremos além para dar suporte à comunidade, ajudando você a contribuir com o ecossistema do Gatsby, oferecendo [brindes para os colaboradores](https://gatsby.dev/swag), dando controle para a comunidade [ao convidar automaticamente todos os colaboradores da organização Gatsby no GitHub](https://github.com/gatsbyjs/gatsby/pull/7699#issuecomment-416665803), um [código de conduta](/contributing/code-of-conduct/) aberto e inclusivo, e outros meios que capacitam e abraçam a incrível comunidade que faz o Gatsby ser possível.

Um dos valores da nossa comunidade é que ["seu lugar é aqui"](/blog/2018-09-07-gatsby-values/#you-belong-here).

> ## Código de Conduta do Colaborador Gatsby

Reviewer: Segundo o guia de tradução do WordPress, "no inglês, a regra de maiúsculas e minúsculas é diferente da usada no português. Em português apenas a primeira palavra da frase e os nomes próprios devem ter a primeira letra maiúscula."

Em alguns títulos você seguiu essa regra e em outros não.

Author: Os títulos estão seguindo o padrão [Title Captalization](https://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html). Alguns estavam fora deste padrão, mas já foram corrigidos.

Reviewer: Acho que deveríamos seguir a regra do português (letra maiúscula apenas na primeira palavra ou em nomes próprios). Mas seria bom ver o que os outros revisores acham.

---
title: Comunidade
---

Desde Setembro de 2019, a comunidade Gatsby é composta por mais de 2.400 colaboradores e dezenas de milhares de desenvolvedores construindo websites pessoais e comerciais usando Gatsby.

> Open source nem sempre tem a melhor reputação de ser amigável e acolhedor, e isso nos deixa tristes. **Todos que pertencem ao open source, e Gatsby, estão dedicados a fazer você se sentir acolhido.**

Reviewer 2: ```suggestion
Open source nem sempre tem a melhor reputação de ser amigável e acolhedor, e isso nos deixa tristes. **Todos que pertencem ao open source, e o Gatsby está dedicado a fazer você se sentir acolhido.**
```

Reviewer 3: Talvez se esse trecho fosse traduzido como:

Open source nem sempre tem a melhor reputação em ser amigável e acolhedor, e isso nos deixa tristes. **Todos fazem parte do open source, e nós do Gatsby, estamos dedicados á fazer você se sentir acolhido.**

A tradução sai um pouco do literal mas talvez deixa mais simples para entender?

> _Open source_ nem sempre tem a melhor reputação de ser amigável e acolhedor, e isso nos deixa tristes. **Todos que pertencem ao _open source_, e Gatsby, estão dedicados a fazer você se sentir acolhido.**

Reviewer 2: ```suggestion
_Open source_ nem sempre tem a melhor reputação de ser amigável e acolhedor, e isso nos deixa tristes. **Todos que pertencem ao _open source_, e todos do Gatsby estão dedicados a fazer você se sentir acolhido.**
```

> Nós nunca julgaremos ou excluíremos alguém. Ao invés disso, nós iremos além para dar suporte a comunidade, ajudando você a contribuir com o ecossistema do Gatsby, oferecendo [brindes para os colaboradores](https://gatsby.dev/swag), dando controle para a comunidade [convidar automaticamente todos os colaboradores da organização Gatsby no GitHub](https://github.com/gatsbyjs/gatsby/pull/7699#issuecomment-416665803), um aberto e inclusivo [código de conduta](/contributing/code-of-conduct/), e outros meios que capacitam e abraçam a incrível comunidade que faz o Gatsby ser possível.

Reviewer 2: ```suggestion
Nós nunca julgaremos ou excluíremos alguém. Ao invés disso, nós iremos além para dar suporte a comunidade, ajudando você a contribuir com o ecossistema do Gatsby, oferecendo [brindes para os colaboradores](https://gatsby.dev/swag), dando controle para a comunidade [convidando automaticamente todos os colaboradores da organização Gatsby no GitHub](https://github.com/gatsbyjs/gatsby/pull/7699#issuecomment-416665803), um aberto e inclusivo [código de conduta](/contributing/code-of-conduct/), e outros meios que capacitam e abraçam a incrível comunidade que faz o Gatsby ser possível.
```

> Nós nunca julgaremos ou excluíremos alguém. Ao invés disso, nós iremos além para dar suporte a comunidade, ajudando você a contribuir com o ecossistema do Gatsby, oferecendo [estilo livre para os colaboradores](https://gatsby.dev/swag), dando controle para a comunidade [convidar automaticamente todos os colaboradores da organização Gatsby no GitHub](https://github.com/gatsbyjs/gatsby/pull/7699#issuecomment-416665803), um aberto e inclusivo [código de conduta](/contributing/code-of-conduct/), e outros meios que capacitam e abraçam a incrível comunidade que faz o Gatsby ser possível.

Reviewer 4: `free swag -> brindes`
Fonte:
https://www.webopedia.com/TERM/S/swag.html

Reviewer 5: Retificado no último commit. ❤️

> Um dos valores da nossa comunidade é: ["você faz parte disso"](/blog/2018-09-07-gatsby-values/#you-belong-here).

Author: Realmente, "aqui é o seu lugar" contextualiza melhor. 
Já mudei na sugestão do @gvarandas.

## Código de conduta do colaborador Gatsby

> No interesse de promover um ambiente aberto e acolhedor, nós como colaboradores e mantenedores juramos fazer da participação no nosso projeto e na nossa comunidade uma experiência livre de assédio, independente da idade, corpo, peso, deficiência, etnia, identidade de gênero e expressão, nível de experiência, nacionalidade, aparência, raça, religião, ou identidade e orientação sexual.

Reviewer 2: ```suggestion
No intuito de promover um ambiente aberto e acolhedor, nós, como contribuidores e mantenedores, nos comprometemos em tornar a participação em nossos projetos e em nossa comunidade o mais livre de assédio para todos, independentemente da idade, corpo, tamanho, deficiência, etnia, identidade ou expressão de gênero, nível de experiência, nacionalidade, aparência, raça, religião ou identidade e orientação sexual..
```
Para ficar igual com o arquivo já traduzido :)

> No intuito de promover um ambiente aberto e acolhedor, nós, como contribuidores e mantenedores, nos comprometemos em tornar a participação em nossos projetos e em nossa comunidade o mais livre de assédio para todos, independentemente da idade, corpo, tamanho, deficiência, etnia, identidade ou expressão de gênero, nível de experiência, nacionalidade, aparência, raça, religião ou identidade e orientação sexual..

Reviewer 3: Apenas ajuste de ponto final:
No intuito de promover um ambiente aberto e acolhedor, nós, como contribuidores e mantenedores, nos comprometemos em tornar a participação em nossos projetos e em nossa comunidade o mais livre de assédio para todos, independentemente da idade, corpo, tamanho, deficiência, etnia, identidade ou expressão de gênero, nível de experiência, nacionalidade, aparência, raça, religião ou identidade e orientação sexual.

Reviewer 5: @Italox você pode fazer uma sugestão de mudança pra ficar mais claro (e mais fácil pro autor corrigir). 🤖

```suggestion
No intuito de promover um ambiente aberto e acolhedor, nós, como contribuidores e mantenedores, nos comprometemos em tornar a participação em nossos projetos e em nossa comunidade o mais livre de assédio para todos, independentemente da idade, corpo, tamanho, deficiência, etnia, identidade ou expressão de gênero, nível de experiência, nacionalidade, aparência, raça, religião ou identidade e orientação sexual.
```

Reviewer 5: Texto retificado no commit mais recente.

[Leia todo o código de conduta](/contributing/code-of-conduct/)

## Formas de contribuir

A comunidade Gatsby está aberta a contribuições. Por favor, consulte os guias abaixo sobre como garantir que suas contribuições sejam aceitas:

- [Como contribuir](/contributing/how-to-contribute/)
- [Como arquivar um problema](/contributing/how-to-file-an-issue/)
- [Criar requisições de novas funções com um RFC](/blog/2018-04-06-introducing-gatsby-rfc-process/)
- [Guia de estilo do Gatsby](/contributing/gatsby-style-guide/)

## Notícias do Gatsby

### Newsletter

> Inscreva-se na newsletter do Gatsby para acompanhar as novidades da comunidade Gatsby. Ouça sobre novas funções, dicas & truques, e o que as pessoas estão construindo.

Reviewer 2: ```suggestion
Inscreva-se na newsletter do Gatsby para acompanhar as novidades da comunidade Gatsby. Ouça sobre novas funções, dicas e truques, e o que as pessoas estão construindo.
```

[Inscreva-se aqui](/newsletter/)

### Twitter

Para últimas notícias sobre Gatsby,
[siga **@gatsbyjs** no Twitter](https://twitter.com/gatsbyjs).

### Encontro da comunidade de mantenedores

A equipe principal do Gatsby (funcionários oficiais que estão trabalhando no open-source do Gatsby) realizam uma reunião semanal onde encorajamos os membros da comunidade a compartilhar os seus projetos, dúvidas e sucessos diretamente com a equipe principal. Além disso, a equipe compartilha informações pertinentes e planos que são aplicáveis a comunidade!

> Esse evento é realizado de forma recorrente, toda Quarta-Feira às 18:30 (Horário padrão do Pacífico).

Reviewer 2: ```suggestion
Esse evento é realizado de forma recorrente, toda Quarta-Feira às 08:30 (Horário padrão do Pacífico).
```
8:30 AM seria 08:30 da manhã

[Adicione aqui o convite recorrente à agenda](https://gatsby.dev/core-maintainers)

## Onde conseguir ajuda

### Stack Overflow

Muitos membros da comunidade usam Stack Overflow para tirar dúvida. Leia as [dúvidas existentes](http://stackoverflow.com/questions/tagged/gatsby) marcadas com `gatsby` ou [faça a sua própria pergunta](http://stackoverflow.com/questions/ask?tags=gatsby)!

### Discord

> Se você precisar de uma resposta imediatamente, confira o [Discord do Gatsby](https://gatsby.dev/discord). Os membros da comunidade e da equipe terão prazer em ajudá-lo ou indicar recursos úteis a você.

Reviewer 5: ```suggestion
Se você precisar de uma resposta imediatamente, confira o [Discord do Gatsby](https://gatsby.dev/discord). Membros da comunidade e da equipe terão prazer em ajudá-lo ou indicar recursos úteis à você.
```

### Hashnode

Muitos membros da comunidade também usam Hashnode para fazer perguntas e compartilhar suas dicas. Leia as [perguntas e postagens existentes](https://hashnode.com/n/gatsby)

> marcado com `gatsby`, [faça sua própria pergunta](https://hashnode.com/create/question) ou [compartilhe sua história](https://hashnode.com/create/story) e adicione a tag `gatsby`.

Reviewer 5: ```suggestion
marcado com `gatsby`, [faça sua própria pergunta](https://hashnode.com/create/question) ou [compartilhe sua história](https://hashnode.com/create/story) e adicione a _tag_ `gatsby`.
```

> marcado com `gatsby`, [faça sua própria pergunta](https://hashnode.com/create/question) ou [compartilhe sua história](https://hashnode.com/create/story) e adicione a tag `gatsby`.

Reviewer 5: ```suggestion
marcadas com `gatsby`, [faça sua própria pergunta](https://hashnode.com/create/question) ou [compartilhe sua história](https://hashnode.com/create/story) e adicione a tag `gatsby`.
```

### DEV Community

> A comunidade DEV também é outra plataforma para fazer perguntas e compartilhar dicas. Leia através de [perguntas e postagens existentes](https://dev.to/t/gatsby) marcado como ´gatsby´ ou [peça ajuda](https://dev.to/new/help). Você também pode [escrever sua postagem](https://dev.to/new/gatsby) com a tag ´gatsby´.

Reviewer 5: ```suggestion
A comunidade DEV também é outra plataforma para fazer perguntas e compartilhar dicas. Leia as [perguntas e postagens existentes](https://dev.to/t/gatsby) marcadas como ´gatsby´ ou [peça ajuda](https://dev.to/new/help). Você também pode [escrever sua postagem](https://dev.to/new/gatsby) com a tag ´gatsby´.
```

> ## Leia mais sobre Gatsby:

Reviewer 5: ```suggestion
## Leia Mais Sobre Gatsby:
```

> - [Por que contribuir para o Gatsby?](/contributing/why-contribute-to-gatsby/)

Reviewer 5: ```suggestion
- [Por Que Contribuir para o Gatsby?](/contributing/why-contribute-to-gatsby/)
```

- [Eventos da Comunidade Gatsby](/contributing/events/)

> - [Onde participar da comunidade?](/contributing/where-to-participate/)

Reviewer 5: ```suggestion
- [Onde participar da Comunidade?](/contributing/where-to-participate/)
```

- [Organize um evento Gatsby](/contributing/organize-a-gatsby-event/)
- [Como dirigir um Workshop Gatsby](/contributing/how-to-run-a-gatsby-workshop/)

> - [How to Pitch Gatsby](/contributing/how-to-pitch-gatsby/)

Reviewer 2: ```suggestion
- [Como apresentar o Gatsby](/contributing/how-to-pitch-gatsby/)
```

Reviewer 2: Acho que é válido esse apontamento

> - [Organize um evento Gatsby](/contributing/organize-a-gatsby-event/)

Reviewer 5: ```suggestion
- [Organize um Evento Gatsby](/contributing/organize-a-gatsby-event/)
```

> - [Como dirigir um Workshop Gatsby](/contributing/how-to-run-a-gatsby-workshop/)

Reviewer 5: ```suggestion
- [Como Organizar um Workshop Gatsby](/contributing/how-to-run-a-gatsby-workshop/)
```

> - [Como apresentar o Gatsby](/contributing/how-to-pitch-gatsby/)

Reviewer 5: ```suggestion
- [Como Apresentar o Gatsby](/contributing/how-to-pitch-gatsby/)
```

> - [Postagens no Blog da comunidade Gatsby](/blog/tags/community/)

Reviewer 5: ```suggestion
- [Postagens no Blog da Comunidade Gatsby](/blog/tags/community/)
```
